### PR TITLE
mergify: automatically apply backend label

### PR DESCRIPTION
**Pull request by russellb**
Automatically apply the `backend` label to any PR that touches a file
under the `docs/backend` directory.

Signed-off-by: Russell Bryant <rbryant@redhat.com>

### CODE REVIEW

File: .github/mergify.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+- name: label-backend
+  description: Automatically apply backend label
+  conditions:
+    - or:
+      - files~=docs/backend/.*
```
Comment (russellb) [inline .github/mergify.yml:6]: I know the `or:` isn't necessary here, but I left it when copying from a different example. I fully expect to add some other paths here soon, like `api-definitions` from #71

**Review by leseb — COMMENTED** [review]
Nit: misspelled "autoamtically".

**Review by nathan-weinberg — APPROVED** [review]
+1 to @leseb's comment but otherwise LGTM

**Comment by russellb** [comment]
> Nit: misspelled "autoamtically".

Thanks -- fixed in the commit message and PR title

**Pull request merged**